STL_Extension/include/CGAL/iterator.h

```diff
@@ -286,7 +303,7 @@ class Counting_output_iterator
 };
 
 template < class I,
-           class Val = typename std::iterator_traits<I>::value_type >
+           class Val = typename CGAL::iterator_traits<I>::value_type >
```

Comment:
Typo

Reply:
Reminder: https://github.com/CGAL/cgal/wiki/Testing#using-ctest

;-)

Reply:
Well, ` std::iterator` still exists. Also I tested.  We maybe have to add a `#define` with the guard of the header of  ` std::iterator`. Or do we have a testsuite where the deprecated no longer exists.

Reply:
- `CGAL::iterator_traits` does not exist, does it?
- `std::iterator_traits` is not deprecated.

Reply:
you are right.